lib/mapper.js

```diff
@@ -47,6 +48,129 @@ exports.track = function(track){
 };
 
 /**
+ * Map Completed Order
+ *
+ * Docs: http://docs.klaviyo.com/article/60-ecommerce-integrating-a-custom-cart
+ *
+ * Must call Top level request for "Placed Order" and separate request for each item in products array
+ * @param {Track} track
+ * @return {Oject}
```

Comment:
How about:

"Returns an object with two properties: .order (Object) and .products (Object[]). Each individual object should be sent in its own call to Klaviyo's API."